src/extension/background-script/connectors/commando.ts

```diff
@@ -261,7 +272,7 @@ export default class Commando implements Connector {
     const transactions: ConnectorTransaction[] = mergeTransactions(
       incomingInvoicesResponse,
       outgoingInvoicesResponse
-    );
+    ).filter((transaction) => transaction.settled);
```

Comment:
is this needed?  because inside `mergeTransactions` there is also filtering? or should that be removed?

Reply:
i added filtering there as well to avoid fallback for settleDate. just a extra check to make typescript happy and function more secure.

Reply:
to me it seems a weird way to do it. Maybe we should remove the filtering in `mergeTransactions` as the function is about merging, not removing elements.

Reply:
resolved